yaptide/routes/utils/tokens.py

```diff
@@ -37,8 +38,22 @@ def encode_auth_token(user_id: int,
         return e, exp
 
 
-def decode_auth_token(token: str,
-                      is_refresh: bool = False) -> Union[int, str]:
+def encode_simulation_auth_token(simulation_id: int):
+    """Function that encodes JWT token for batched simulation 'update_key'"""
```

Comment:
by batch we mean submission to Ares HPC, by celery we mean celery queue on local instance. 
This function seems to be used in both scenarios, is that true ?
Adjust the docstring

Reply:
Don't know what was the intention here. I'll edit docstring as sugested